messages/lightning.preview.app.md

```diff
@@ -29,11 +27,11 @@ Username or alias of the target org. Not required if the `target-org` configurat
 
 # flags.device-type.summary
 
-Type of device to emulate in preview.
+Type of device to emulate in the preview.
```

Comment:
Independent of your change here, we had some discussion about changing the "emulate" verbiage because it was a bit misleading. This is what we came up with. Does this work for you?

```suggestion
Type of device to be used for previewing the content.
```

Reply:
@khawkins Would something like this be equivalent? 
```suggestion
Type of device to load the app preview.
```

I'm not sure if "load" is an OK verb -- maybe "render" or "run?" We could also do "use," like "Type of device to use for the app preview."

Reply:
I think "load" could work. Maybe "display" could work as well? I'm good to stick with "load", until someone tells me otherwise.

Reply:
I like display!